homeworks/01-bubble-sort/src/components/app/App.tsx

```diff
@@ -0,0 +1,166 @@
+import React from 'react';
+import Field from '../field/field';
+import {randomArray, delay} from "../../assets/utils";
+
+// styles
+import './app.scss';
+
+enum Status {
+  notStarted = "not started",
+  started = "started",
+  paused = "paused",
+  solved = "solved"
```

Comment:
PascalCase is more preferable for enum keys names.
![image](https://user-images.githubusercontent.com/24900675/115388059-1dd44c80-a1e4-11eb-817e-dcc576084076.png)